test/rules/comment-format/upper/test.ts.lint

```diff
@@ -10,7 +10,8 @@ class Clazz { // This comment is correct
         console.log("test"); //This comment has no space
                                ~~~~~~~~~~~~~~~~~~~~~~~~~ [space]
     }
-    /// <reference or something>
+    /// <not a reference>
+      ~~~~~~~~~~~~~~~~~~~ [space]
```

Comment:
Yeah, it should handle comments starting with more than 2 slashes better. The comment text should only be checked starting from the first non-slash character. 

And maybe there could be a new option to disallow more than 2 slashes